txndata/constraints.lisp

```diff
@@ -323,92 +323,90 @@
-                         (result-must-be-true  comparison---maxfee-and-max-priority-fee-row-offset)))
+                  (begin (small-call-to-LEQ    row-offset---max-fee-and-max-priority-fee-comparison (max_priority_fee) (max_fee))
+                         (result-must-be-true  row-offset---max-fee-and-max-priority-fee-comparison)))
 
 (defconstraint    comparison-for-type-2---computing-the-effective-gas-price         (:guard    (*   (first-row-of-new-transaction)   TYPE2))
-                  (small-call-to-LEQ   comparison---computing-effective-gas-price-row-offset (+ (max_priority_fee) BASEFEE) (max_fee)))
+                  (small-call-to-LEQ   row-offset---computing-effective-gas-price-comparison (+ (max_priority_fee) BASEFEE) (max_fee)))
 
-(defun (get_full_tip) (force-bool (shift RES comparison---computing-effective-gas-price-row-offset)))
+(defun (get_full_tip) (force-bool (shift RES row-offset---computing-effective-gas-price-comparison)))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;                                    ;;
 ;;    2.11 Setting certain variables  ;;
 ;;                                    ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(defconstraint setting-gas-init (:guard (first-row-of-new-transaction))
-               (if-not-zero TYPE0
-                            (eq! GAS_INITIALLY_AVAILABLE (- (gas_limit) (legacy_upfront_gas_cost)))
-                            (eq! GAS_INITIALLY_AVAILABLE (- (gas_limit) (access_upfront_gas_cost)))))
+(defconstraint   setting-gas-initially-available (:guard (first-row-of-new-transaction))
+                 (if-not-zero TYPE0
+                              (eq! GAS_INITIALLY_AVAILABLE (- (gas_limit) (legacy_upfront_gas_cost)))
+                              (eq! GAS_INITIALLY_AVAILABLE (- (gas_limit) (access_upfront_gas_cost)))))
 
-(defconstraint setting-gas-price (:guard (first-row-of-new-transaction))
-               (if-zero TYPE2
-                        (eq! GAS_PRICE (gas_price))
-                        (if-not-zero (get_full_tip)
-                                     (eq! GAS_PRICE (+ BASEFEE (max_priority_fee)))
-                                     (eq! GAS_PRICE (max_fee)))))
+(defconstraint   setting-gas-price (:guard (first-row-of-new-transaction))
+                 (if-zero TYPE2
+                          (eq! GAS_PRICE (gas_price))
+                          (if-not-zero (get_full_tip)
+                                       (eq! GAS_PRICE (+ BASEFEE (max_priority_fee)))
+                                       (eq! GAS_PRICE (max_fee)))))
 
-(defconstraint setting-priority-fee-per-gas (:guard (first-row-of-new-transaction))
-               (if-zero TYPE2
-                        (eq! PRIORITY_FEE_PER_GAS (gas_price))
-                        (eq! PRIORITY_FEE_PER_GAS (- GAS_PRICE BASEFEE))))
+(defconstraint   setting-priority-fee-per-gas (:guard (first-row-of-new-transaction))
+                 (eq! PRIORITY_FEE_PER_GAS (- GAS_PRICE BASEFEE)))
```

Comment:
@benSepanski This is where we set the `PRIORITY_FEE_PER_GAS` regardless of the transaction type.